example-data.sql

```diff
@@ -46,7 +46,7 @@ INSERT INTO datapoint (id,variable_id,runid,timestamp,value) VALUES (15,13,2,'20
 INSERT INTO datapoint (id,variable_id,runid,timestamp,value) VALUES (16,13,3,'2020-07-30 12:00:54.678',205);
 INSERT INTO change(id, confirmed, description, variable_id, runid, timestamp) VALUES (17, false, 'Test annotation <a href="http://localhost:3000/" target="_parent">with a link</a>', 13, 3, '2020-07-30 12:00:54.678');
 
-INSERT INTO watch(id, testid) VALUES (18, 2);
+INSERT INTO watch(id, testid, mutemissingruns) VALUES (18, 2, false);
```

Comment:
Shouldn't this be just `missingruns`?

TBH it seems to me that `missingruns = true` is not very clear name for the fact that the missing runs notifications are to be **muted** (I would expect the opposite meaning). Maybe at least add a comment to the entity if you prefer it this way.

Reply:
Yes it should be. Thanks for spotting that.
 Changing the field name made sense when it came to the UI Switch label. Writing a label that was obvious sense was not easy. A double negative. ie, "turn it on" to cease notifications.
 I've changed it to default to true. So that users can see they need to "turn it off" to cease notifications.